packages/python-google-compute-engine/google_compute_engine/instance_setup/tests/instance_setup_test.py

```diff
@@ -421,6 +428,27 @@ def testSetupBotoConfigLocked(self, mock_boto):
     instance_setup.InstanceSetup._SetupBotoConfig(self.mock_setup)
     self.mock_logger.warning.assert_called_once_with('Test Error')
 
+  @mock.patch('google_compute_engine.instance_setup.instance_setup.subprocess')
+  def testDisableOvercommitNonE2(self, mock_subprocess):
+    self.mock_setup.metadata_dict = {
+        'instance': {
+            'machineType': 'projects/00000000000/machineTypes/n1-standard-1'
```

Comment:
Please add a comma to the end of the line since it is an entry in the dictionary. (Also apply to the test case below).